docs/getting-started/integrations-cdn/vuejs-v3.md

```diff
@@ -8,150 +8,114 @@ order: 50
 
 {@snippet installation/integrations/framework-integration}
 
-# Vue.js 3+ rich text editor component
+# Vue.js 3+ rich text editor component (CDN)
 
 <p>
 	<a href="https://www.npmjs.com/package/@ckeditor/ckeditor5-vue" target="_blank" rel="noopener">
 		<img src="https://badge.fury.io/js/%40ckeditor%2Fckeditor5-vue.svg" alt="npm version" loading="lazy">
 	</a>
 </p>
 
-Vue.js is a versatile framework for building web user interfaces. CKEditor&nbsp;5 provides the official Vue component you can use in your application.
+CKEditor&nbsp;5 has an official Vue integration that you can use to add a rich text editor to your application. This guide will help you install it and configure to use the CDN distribution of the CKEditor&nbsp;5.
 
-<info-box hint>
-	Starting from version 5.0.0 of this package, you can use native type definitions provided by CKEditor&nbsp;5. Check the details about {@link getting-started/setup/typescript-support TypeScript support}.
-</info-box>
+This guide assumes that you already have a Vue project. If you do not have one, see the [Vue documentation](https://vuejs.org/guide/quick-start) to learn how to create it.
 
 ## Quick start
 
 {@snippet getting-started/use-builder}
 
-### Setting up the project
-
-This guide assumes that you already have a Vue project. If you do not have one, see the [Vue documentation](https://vuejs.org/guide/quick-start) to learn how to create it.
-
-### Installation
-
-Start by installing the following packages:
-
-* `ckeditor5` &ndash; contains all open-source plugins and features for CKEditor&nbsp;5.
-
-	```bash
-	npm install ckeditor5
-	```
-
-* `ckeditor5-premium-features` &ndash; contains premium plugins and features for CKEditor&nbsp;5. Depending on your configuration and chosen plugins, you might not need it.
-
-	```bash
-	npm install ckeditor5-premium-features
-	```
-
-* `@ckeditor/ckeditor5-vue` &ndash; the [CKEditor&nbsp;5 WYSIWYG editor component for Vue](https://www.npmjs.com/package/@ckeditor/ckeditor5-vue).
-
-	```bash
-	npm install @ckeditor/ckeditor5-vue
-	```
-
-With these packages installed, you now need to choose whether to install the `<ckeditor>` component globally or locally and follow the appropriate instructions below.
-
-#### Installing the `<ckeditor>` component globally
-
-To register the `<ckeditor>` component globally, you must install the CKEditor&nbsp;5 plugin for Vue.
-
-If you are using a plain Vue project, you should find the file where the `createApp` function is called and register the `CkeditorPlugin` plugin with the [`use()` method](https://vuejs.org/api/application.html#app-use).
+### Installing and configuring the Vue integration
```

Comment:
The heading is a bit long. Maybe we should divide it into two:

1. Installing the Vue integration from npm
2. Configuring the integration

The headings are shorter in the table of contents, and it's unified with the structure of other guides.

```suggestion
### Installing the Vue integration from npm
```

Reply:
I'm not a fan of having an entire separate section for an `npm install`